site/app/templates/polls/NewPollPage.twig

```diff
@@ -6,14 +6,14 @@
             <input type="hidden" name="csrf_token" value="{{ csrf_token }}"/>
             <input type="hidden" name="response_count" id="response-count" value="0"/>
 
+            <span class="option-title" id="poll-name-label">Poll Name:</span>
             <label for="poll-name">
```

Comment:
The label should be around the text ("Poll Name:") that the input belongs to for accessibility. Wrapping it around the input itself does not do anything.

See: https://www.w3.org/WAI/tutorials/forms/labels/

This applies to the other changes within this file.